### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MayerLansky**
[DEMO LINK](https://MayerLansky.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://MayerLansky.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
your solution doesn't match the design at all
fix it (look on tests)
feel free to ask in `fe_chat`
also go through [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) before sending your code

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,37 @@
...
+            <li class="links"><a href="#">TABLETS</a></li>
+            <li class="links"><a href="#">PHOTO</a></li>
+            <li class="links"><a href="#">VIDEO</a></li>
+          </ul>
+        </label>
+      </nav>   
+      
+    </header>
+    
   </body>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:37]:
```suggestion
      </nav>   
    </header>
  </body>
```

File: src/index.html [hunk 2]
```diff
@@ -2,13 +2,37 @@
...
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          class="logo" 
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav_list">
+        <label>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:21]: why do you need label here?

File: src/index.html [hunk 3]
```diff
@@ -2,13 +2,37 @@
...
+          <ul>
+            <li class="links"><a href="#" class="is-active">APPLE</a></li>
+            <li class="links"><a href="#">SAMSUNG</a></li>
+            <li class="links"><a href="#">SMARTPHONES</a></li>
+            <li class="links" data-qa="hover"><a href="#">LAPTOPS & COMPUTERS</a></li>   <!-- data hover, is it right?-->
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:26]: it is, remove comment

File: src/style.css [hunk 4]
```diff
@@ -1 +1,83 @@
...
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  height: 100vh;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:12]: you don't need it

File: src/style.css [hunk 5]
```diff
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:7]: do you have default padding at body?

File: src/style.css [hunk 6]
```diff
@@ -1 +1,83 @@
...
+  justify-content: space-between;
+  flex-direction: row;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+  
+.nav_list {
+  list-style: none;
+  margin-right: 50px;
+  margin-left: 509px;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:27]: do not hardcode this values - your layout should be responsive

File: src/style.css [hunk 7]
```diff
@@ -1 +1,83 @@
...
+.links {
+  display: inline-block;
+  font-family: inherit;
+  text-align: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+  
+a {
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:40]: use class selectors

File: src/style.css (hunk 6 above)
Comment (Bohdan-Maksymiuk) [reply src/style.css:27]: consider adding horizontal paddings for header

File: src/style.css [hunk 8]
```diff
@@ -1 +1,83 @@
...
+  text-transform: uppercase;
+  color: #000;
+  padding: 23px 0;
+  margin: 0 20px;
+  line-height: 60px;
+  cursor: pointer;
+}
+  
+.logo {
+  position: absolute;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:51]: are you sure that you need absolute here?

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Screenshot 2022-11-17 at 14 00 04

your solution doesn't match design at all - ask in chat or visit q&a

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Links are not working

**Comment by MayerLansky** [comment]
What should I do? Last time they worked

**Comment by MayerLansky** [comment]
[DEMO LINK](https://MayerLansky.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://MayerLansky.github.io/layout_moyo-header/report/html_report/)

**Pull request closed without merging**